precompiles/assets-erc20/src/eip2612.rs

```diff
@@ -191,21 +191,27 @@ where
 	) -> EvmResult<PrecompileOutput> {
 		handle.record_cost(RuntimeHelper::<Runtime>::db_read_gas_cost())?;
 
-		let mut input = handle.read_input()?;
-		let owner: H160 = input.read::<Address>()?.into();
-		let spender: H160 = input.read::<Address>()?.into();
-		let value: U256 = input.read()?;
-		let deadline: U256 = input.read()?;
-		let v: u8 = input.read()?;
-		let r: H256 = input.read()?;
-		let s: H256 = input.read()?;
+		read_args!(
+			handle,
+			{
```

Comment:
This was modified before no? you should probably merge master to get an idea on the real changes

Reply:
Yes I worked on top of the other PR to have its features.